JPS_Ontology/ontology/ontoreaction/OntoReaction.owl

```diff
@@ -130,8 +180,8 @@
     <owl:ObjectProperty rdf:about="https://www.theworldavatar.com/kg/ontoreaction/hasPerformanceIndicator">
         <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontoreaction/ReactionExperiment"/>
         <rdfs:range rdf:resource="https://www.theworldavatar.com/kg/ontoreaction/PerformanceIndicator"/>
-        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A relation between a reaction experiment and the performance indicator calculated from the measured analytical data of its output chemical. </rdfs:comment>
-        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoreaction/</rdfs:isDefinedBy>
+        <rdfs:comment>A relation between a reaction experiment and the performance indicator calculated from the measured analytical data of its output chemical. </rdfs:comment>
```

Comment:
Seems the way how `rdfs:comment` and `rdfs:isDefinedBy` are represented in the owl file is changed throughout, is this something resulting from the updates in TBoxGenerator?

Reply:
Yes. I am using the latest version of the tbox generator from the main branch.